CONTRIBUTING.md

```diff
@@ -29,3 +29,126 @@ Recommendations and requirements for how to best contribute to Parquet. We striv
 ### License
 By contributing your code, you agree to license your contribution under the terms of the APLv2:
 https://github.com/apache/parquet-format/blob/master/LICENSE
+
+### Additions/Changes to the Format
+
+Note: This section applies to actual functional changes to the specification.
+Fixing typos, grammar, and clarifying concepts that would not change the
+semantics of the specification can be done as long a comitter feels comfortable
+to merge them. When in doubt starting a discussion on the dev mailing list is
+encouraged.
+
+The general steps for adding features to the format are as follows:
+
+1. Discuss changes on on the developer mailing list (dev@parquet.apache.org).
+   Often times it is helpful to link to a draft pull request to make the
+   discussion concrete. This step is complete when there lazy consensus. Part
+   of the consensus is whether it sufficient to provide 2 working
+   implementations as outlined in step 2 or if demonstration of the feature
+   with a down-stream query engine is necessary to justify the feature (e.g.
+   demonstrate performance improvements in Arrow's DataSet library or
+   Apache Data Fusion).
+
+2. Once a change has lazy consensus two implementations of the feature
```

Comment:
One thing that's a little confusing to me is the sequencing - is the expectation that the 2 implementations implement the feature first before it's introduced into the standard? Or is it expected that the format is released with 2 implements "verbally" (or over email) committing to implement the new feature?

Reply:
I'll clarify my suggestion (and what is common for new features recently) is two open PRs for the implementations, so more then a verbal commitment

Reply:
This is clarified now.